x-pack/plugins/security/server/lib/authentication/providers/basic.js

```diff
@@ -9,6 +9,37 @@ import { canRedirectRequest } from '../../can_redirect_request';
 import { AuthenticationResult } from '../authentication_result';
 import { DeauthenticationResult } from '../deauthentication_result';
 
+/**
+ * Utility class that knows how to decorate request with proper Basic authentication headers.
+ */
+export class BasicCredentials {
```

Comment:
Super annoying, but the linter has a rule that disallows accessing static methods on BasicCredentials before it is defined in the same file. Even though it's all static. Dumb.

Anyway, I copied and pasted `BasicCredentials` without changes to the top of the file so linting would stop failing